css/doc.css

```diff
@@ -1,58 +1,57 @@
-@import url(nav.css);
+@import url(css\nav.css);
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
 /* Docs */
 .main{
     display: flex;
     flex: 0 0 100%;
-    gap: 50px;
+    gap: 3rem;
  } 
 .left-container{
     width: 15%;
 }
 .gutter{
-    margin: 0px 30px 20px 140px;
+    margin: 0rem 1.8rem 1.25rem 8.75rem;
 }
-.item{
+.list-menu{
     list-style: none;
     margin-top: 10px;
     padding: 10px;
     cursor: pointer;
 }
-.item:hover{
+.list-menu:hover{
     background: var(--primary-color);
     border-radius: 5px;
 }
-.item:focus{
+.list-menu:focus{
     color: var(--seconday-color);
 }
-.item-one:hover{
+.list-menu-no-hover:hover{
     background: none;
-    border-radius: 0px;
+    border-radius: 0rem;
 }
-.item a{
+.list-menu a{
```

Comment:
Instead of selecting the anchor or a tag within list-menu ,You can rename this class as -list-menu-link and then pass the css properties related to it. and then you can pass this class within that HTML file.

Reply:
I was supposed to use this on anchor tags of my whole project. That's why I used it. From next time I will first make class and then declare css properties.

Reply:
Alright!!Understood